dizoo/evogym/envs/test/visualize_simple_env.py

```diff
@@ -0,0 +1,34 @@
+import gym
+from evogym import sample_robot
```

Comment:
It would be better to give a proper order for modules that imported.

Reply:
this file will be eliminated, it is to debug the video function that is giving some problems